api-report/container-loader.api.md

```diff
@@ -128,6 +125,8 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     get resolvedUrl(): IResolvedUrl | undefined;
     // (undocumented)
     resume(): void;
+    // Warning: (ae-forgotten-export) The symbol "IConnectionArgs" needs to be exported by the entry point index.d.ts
```

Comment:
It's interesting that this works (`resumeInternal` with `IConnectionArgs` arguments) when `IConnectionArgs` is no longer exported. Should that interface still be exported?

Reply:
This does mean that the `IConnectionArgs` interface needs to be exported by this module. You can use `export type` if you want to be super-safe. <https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export>